linkml/generators/golrgen.py

```diff
@@ -84,7 +84,7 @@ def visit_class_slot(self, cls: ClassDefinition, aliased_slot_name: str, slot: S
 
 @shared_arguments(GolrSchemaGenerator)
 @click.command()
-@click.option("--dir", "-d", default='golr-views', help="Output directory")
+@click.option("--dir", "-d", default='golr-views', show_default=True, help="Output directory")
```

Comment:
is this meaningful? There is no default defined

Reply:
Oh there is, isn't there? The default value, in this case for the directory name has been set to `golr-views`.

Reply:
oh you are quite right,I misread!